src/smc-webapp/frame-editors/frame-tree/editor.tsx

```diff
@@ -43,7 +42,7 @@ interface FrameTreeEditorReduxProps {
   errorstyle: ErrorStyles;
   cursors: Map<string, any>;
   status: string;
-  load_time_estimate?: Map<string, any>;
+  load_time_estimate?: LoadingEstimate;
```

Comment:
`Map<string, any>;` is just not what the component gets. There is the type `LoadingEstimate`  now. I hope this is fine here?

Reply:
LoadingEstimate **is a** Map<string, any>, so this is fine, since LoadingEstimate is a TypedMap (which is just a special Map<string, any>).  You just replaced one valid type with a vastly better valid type.